lib/active_model_serializers/adapter/json_api/pagination_links.rb

```diff
@@ -2,14 +2,20 @@ module ActiveModelSerializers
   module Adapter
     class JsonApi < Base
       class PaginationLinks
+        MissingSerializationContextError = Class.new(KeyError)
         FIRST_PAGE = 1
 
         attr_reader :collection, :context
 
         def initialize(collection, adapter_options)
           @collection = collection
           @adapter_options = adapter_options
-          @context = adapter_options.fetch(:serialization_context)
+          @context = adapter_options.fetch(:serialization_context) do
+            fail MissingSerializationContextError, 'JsonApi::PaginationLinks requires a'\
```

Comment:
just because I have things in the others that need double quotes. rubocop caught it.  changing to a heredoc